test/module/irohad/torii/torii_service_test.cpp

```diff
@@ -50,70 +95,45 @@ class ToriiServiceTest : public testing::Test {
     th.join();
   }
 
-  ServerRunner* runner;
+  ServerRunner *runner;
   std::thread th;
 };
 
 TEST_F(ToriiServiceTest, ToriiWhenBlocking) {
   for (size_t i = 0; i < TimesToriiBlocking; ++i) {
-    std::cout << i << std::endl;
     iroha::protocol::ToriiResponse response;
-    auto stat = torii::CommandSyncClient(Ip, Port)
-      .Torii(iroha::protocol::Transaction {}, response);
+    // One client is generating transaction
+    auto new_tx = iroha::protocol::Transaction();
+    auto meta = new_tx.mutable_meta();
+    meta->set_tx_counter(i);
+    meta->set_creator_account_id("accountA");
+    auto stat = torii::CommandSyncClient(Ip, Port).Torii(new_tx, response);
     ASSERT_TRUE(stat.ok());
-    std::cout << "Sync Response\n";
+    ASSERT_EQ(response.validation(),
+              iroha::protocol::STATELESS_VALIDATION_SUCCESS);
   }
 }
 
 TEST_F(ToriiServiceTest, ToriiWhenNonBlocking) {
```

Comment:
Not obvious what testing this method. May be provide some documentation for that?